src/events/SDL_mouse.c

```diff
@@ -173,6 +173,24 @@ static void SDLCALL SDL_MouseTouchEventsChanged(void *userdata, const char *name
     }
 }
 
+static void SDLCALL SDL_PenMouseEventsChanged(void *userdata, const char *name, const char *oldValue, const char *hint)
+{
+    SDL_Mouse *mouse = (SDL_Mouse *)userdata;
+
+    mouse->pen_mouse_events = SDL_GetStringBoolean(hint, true);
+}
+
+static void SDLCALL SDL_PenTouchEventsChanged(void *userdata, const char *name, const char *oldValue, const char *hint)
+{
+    SDL_Mouse *mouse = (SDL_Mouse *)userdata;
+
+    mouse->pen_touch_events = SDL_GetStringBoolean(hint, true);
+
+    if (mouse->pen_touch_events) {
```

Comment:
This should probably protect against double sets and clear the touch device if this is turned off.

Reply:
Hmm...the mouse code doesn't do that either, though.  :/

Reply:
Hmm, okay, maybe check in as-is and we can add a bug to fix both of them.

Reply:
Okay, I'm blocked on #11970, but I can clean all this up after.

Reply:
This is cleaned up in ff3442138c083b1cea23cf3e452d7cc3e289a505.